libs/libc/misc/lib_ioctl.c

```diff
@@ -102,7 +102,7 @@ int ioctl(int fd, int req, ...)
    * discover cases where something worse happens!
    */
 
-  DEBUGASSERT(sizeof(int)        == sizeof(unsigned long) &&
+  DEBUGASSERT(sizeof(int)        <= sizeof(unsigned long) &&
```

Comment:
Thinking more about this, I think we should just remove this check.  sizeof(int) is guanteed to C requirements to be less than or equal to sizeof(unsigned long).  So this is just checking if the compiler is sane.  It is not checking anything of value..